Exercises/multi_gpu_cuda/Begin/CMakeLists.txt

```
@@ -1,7 +1,15 @@
 cmake_minimum_required(VERSION 3.16)
 project(KokkosTutorialMultiGpuCuda)
-include(../../common.cmake)
-  
-add_executable(ExerciseMultiGPU multi_gpu_cuda.cpp)
-target_link_libraries(ExerciseMultiGPU Kokkos::kokkos)
 
+# Add a custom module path for find_package
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/../../.cmake)
+
+find_package(Kokkos REQUIRED)
+
+if (NOT Kokkos_DEVICES MATCHES "CUDA")
```

```suggestion
if (NOT (Kokkos_DEVICES MATCHES "CUDA" OR Kokkos_ENABLE_CUDA))
```

Otherwise the warning message is always displayed when Kokkos is retrieved by FetchContent.

---

Good catch!

Looks like `Kokkos_ENABLE_CUDA` is always working, so I'll skip the `Kokkos_DEVICES` test.